seps/sep-0002.md

```diff
@@ -0,0 +1,225 @@
+
+
+1. devs.near's groups [defined implicitly by follow-style graph connections](https://near.org/devs.near/widget/every.group). 
+
+A primary drawback to this graph-based approach is that there is no canonical source of group membership. A single group of N members can have M distinct versions of who belongs in that group, where M is the number of group members who decline to "mutually follow" each newly requested group member.  Group membership is managed by an object on the namespace of anyone who has requested access to a group, which contains a group identifer as a key with a value of a list of accountIds (AccountList). Even if no one approves someone's request to join a group, the requestor would retain a valid instance of that groupIdentifer:AccountList on their socialDB namespace - with their accountID included as a group member. This results in muliple versions of graphs which represent the definition of members of a group. For more details on this implementation see minute 33:45 [here](https://youtu.be/HAyweTF7IjU?si=z_hfct3oK_86nt_o)
+
+Which this approach, can we efficiently determine which group members should receive a notification when a post is created on that group's page?  How challenging would it be to ensure that only group members can posts to this group's page? There is a solution here, however these implicitly defined group connections with multiple versions of group membership adds complexity. 
+
+
+1. [Communities on DevHub](https://near.org/devgovgigs.near/widget/Ideas)
+
+DevHub's community implementation exists on a distint contract from near.social. This results in a lack of discoverability of DevHub's groups across gateways due to its community data not being included in the indexes which power search on socialdB.
+
+
+## Specification 
+
+1. Adjacent to root level socialDB accounts, create a new top account called `groups.near`. However, since this account name has already been claimed, I have slightly rearranged the same letters to use the spanish translation: `grupos.near`. For convience, gateway operators could add a redirect from `gatway.com/near/groups.near` -> `gatway.com/near/grupos.near` if desired.
```

Comment:
@thisisjoshford please elaborate

Reply:
I just mean we have TLA accounts like `aurora`, `near`, etc. and could use simply `groups` over `groups.near`

Reply:
I suspect that it would be difficult to justify the introduction of `groups` as a TLA because we don't entirely need unique accounts for groups. Also this would introduce the problem (benefit?) of determining how to deal with unique group names within this domain. Would you recommend `[groupName].groups`? Then we would need to deal with squatters sitting on cool group names like `theopenweb.groups`.  Alternatively we could using unique identifiers like the groupUID mentioend in this SEP, but I think `00110357asiedjf0935t.groups` is unsightly.